.github/workflows/phpunit.yml

```diff
@@ -88,7 +88,7 @@ jobs:
       # do not stop on another job's failure
       fail-fast: false
       matrix:
-        php-versions: ['8.0', '8.1', '8.2']
+        php-versions: ['8.0', '8.1', '8.2', '8.3']
```

Comment:
works, but bloats the matrix. could you move 8.0 out of this array and have one dedicated job in the `include` section below?